docs/topics/DatabaseOperations.adoc

```diff
@@ -85,6 +105,26 @@ To edit the details in an entry, perform the following steps:
 
 4. Click *OK*.
 
+=== Adding TOTP to an Entry
+Timed One-Time Passwords (TOTP) are a popular choice for two-factor authentication methods. These codes are typically six digits long and change every 30 seconds. They are derived from a shared secret value and the current time. Once set up, KeePassXC will calculate the current value of a TOTP code for use with copy/paste, browser extension, and Auto-Type.
+
+WARNING: Your computer time must be synchronized with an internet time source to generate valid TOTP codes, https://www.nist.gov/pml/time-and-frequency-division/time-distribution/internet-time-service-its[read more here].
+

```

Comment:
Maybe we should add a note/warning that storing a TOTP secret with your password in the same database defeats the purpose of it, even when it's convenient?

Reply:
I have had that thought too. Would it be possible / sensible to implement two secure stores inside a keepassxc file, ie. One for passwords as usual and a separate one which only contains the totp secrets, which must be unlocked by a separate unrelated password to be used ?